modules/roomify/roomify_listing/roomify_listing.fields.inc

```diff
@@ -2645,43 +2645,46 @@ function roomify_listing_create_standard_type_fields($type_bundle) {
   }
 
   if (field_read_instance('bat_type', 'field_st_allow_instant_bookings', $type_bundle) === FALSE) {
-    $field_instances['bat_type-standard_type-field_st_allow_instant_bookings'] = array(
-      'bundle' => $type_bundle,
-      'default_value' => array(
-        0 => array(
-          'value' => 1,
+    if ($type_bundle == 'room') {
+      // Add the instant bookings only for rooms.
+      $field_instances['bat_type-standard_type-field_st_allow_instant_bookings'] = array(
+        'bundle' => $type_bundle,
+        'default_value' => array(
+          0 => array(
+            'value' => 1,
+          ),
         ),
-      ),
-      'description' => 'Enabling instant bookings means that users will be able to book and pay immediately. Make sure to setup a suitable payment method.',
-      'display' => array(
-        'default' => array(
-          'label' => 'above',
-          'module' => 'list',
-          'settings' => array(
-            'conditions' => array(),
+        'description' => 'Enabling instant bookings means that users will be able to book and pay immediately. Make sure to setup a suitable payment method.',
```

Comment:
@acrollet I added a patch to the Token module to solve this known issue. Now label and descriptions of fields are completely translatable :)